samples/detect.v1p4beta1.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * Copyright 2019, Google,LLC.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+
+'use strict';
+
+async function detectBatchAnnotateFiles(fileName) {
+  // [START vision_batch_annotate_files_beta]
+  // Imports the Google Cloud client libraries
+  const vision = require('@google-cloud/vision').v1p4beta1;
+  const fs = require('fs');
+
+  // Creates a client
+  const client = new vision.ImageAnnotatorClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const fileName = `/path/to/localDocument.pdf`;
+
+  const inputConfig = {
+    // Other supported mime_types: image/tiff' or 'image/gif'
+    mimeType: 'application/pdf',
+    content: fs.readFileSync(fileName),
```

Comment:
folks copy/paste these samples, and using `readFileSync` is really bad for performance in nodejs.  Instead, can you please use `util.promisify` to create a promisied version of `fs.readFile` and `await` it?

Reply:
Sure, let me figure out what the change should be.

Reply:
Done